raisinbread/RB_files/RB_user_perm_rel.sql

```diff
@@ -26,8 +26,6 @@ INSERT INTO `user_perm_rel` (`userID`, `permID`) VALUES (1,22);
 INSERT INTO `user_perm_rel` (`userID`, `permID`) VALUES (1,23);
 INSERT INTO `user_perm_rel` (`userID`, `permID`) VALUES (1,24);
 INSERT INTO `user_perm_rel` (`userID`, `permID`) VALUES (1,25);
-INSERT INTO `user_perm_rel` (`userID`, `permID`) VALUES (1,26);
-INSERT INTO `user_perm_rel` (`userID`, `permID`) VALUES (1,27);
 INSERT INTO `user_perm_rel` (`userID`, `permID`) VALUES (1,28);
```

Comment:
```suggestion
INSERT INTO `user_perm_rel` (`userID`, `permID`) VALUES (1,26);
```

Reply:
@AlexandraLivadas You need to re-order the permIDs after you remove the 26 and 27.